src/helpers/fallbackauthdialog.ts

```diff
@@ -44,23 +49,18 @@ const msalConfig: Msal.Configuration = {
   },
 };
 
-var requestObj: Object = {
+var requestObj: RedirectRequest = {
   scopes: [`https://graph.microsoft.com/User.Read`],
 };
 
-const userAgentApp: Msal.UserAgentApplication = new Msal.UserAgentApplication(msalConfig);
+const publicClientApp: PublicClientApplication = new PublicClientApplication(msalConfig);
 
-function authCallback(error, response) {
-  if (error) {
-    console.log(error);
-    Office.context.ui.messageParent(JSON.stringify({ status: "failure", result: error }));
+function handleResponse(response) {
+  if (response.tokenType === "id_token") {
+    console.log(response.idToken.rawIdToken);
```

Comment:
Don't log out the token as it contains PII.

Reply:
BTW . . . the sso sample you've been working on also logs this out :-).

Reply:
It shouldn't, if you can point out where that is happening, would be super useful and I will fix it! Thanks!

Reply:
authRedirect.js L24

Reply:
Actually . . . I was confusing two calls . . . please disreguard.